ceph/utils.py

```diff
@@ -107,9 +107,15 @@ def create_ceph_nodes(
 
                 if node_dict.get("cloud-data"):
                     node_params["cloud-data"] = node_dict.get("cloud-data")
-
+                node_count += 1
                 p.spawn(setup_vm_node, node, ceph_nodes, **node_params)
 
+    if len(ceph_nodes) != node_count:
+        log.error(
```

Comment:
Would it be possible to use `string interpolation` instead? For example:

```
log.error("Mismatch error in number of VMs spawned. Expected: {len(ceph_nodes)} \t Actual: {node_count}")
```

Reply:
Done